app/src/main/java/com/example/demouniverload/search/presenter/MethodsOfYandexParser/YandexImageParser.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demouniverload.search.presenter.MethodsOfYandexParser;
+
+import com.example.demouniverload.search.presenter.ParsersInterface;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+import java.util.ArrayList;
+
+public class YandexImageParser implements ParsersInterface.YandexMethodsInterface.IYandexImageParser {
+    @Override
+    public ArrayList<String> parseLinksOfElements(Document document) {//Parse Links of element
```

Comment:
```suggestion
    public ArrayList<String> parseLinksOfElements(Document document) {
```
This comment is pretty useless. It repeats the method's name.